Marlin/src/feature/bedlevel/ubl/ubl.cpp

```diff
@@ -286,9 +286,11 @@ void GcodeSuite::M1004() {
     process_subcommands_now_P(PSTR ("G28"));
     process_subcommands_now_P(PSTR (ALIGN_GCODE));
     process_subcommands_now_P(PSTR (PROBE_GCODE));
+    process_subcommands_now_P(PSTR ("G29P3\nG29P3"));
```

Comment:
Should this be "`G29P3\nG29P3`" or "`G29P3\nM500`"?

Reply:
these 3 additional G29 P3 are a just in case the user has selected a large grid like 15x15 on a small bed resulting in 3 missing rows.